azavea/loam-geotiff-info
# Style pass

# Overview
- Styles Loam project
- Adds Azavea favicons
- Incorporates Adobe fonts
- Basemap style
- Adds Google Analytics

# Demo
<img width="1792" alt="Screen Shot 2021-02-22 at 3 22 32 PM" src="https://user-images.githubusercontent.com/5672295/108702057-2bda5980-7522-11eb-9419-7e150448d45f.png">
<img width="1792" alt="Screen Shot 2021-02-22 at 3 43 11 PM" src="https://user-images.githubusercontent.com/5672295/108703914-b58b2680-7524-11eb-8cbc-828924ab63b9.png">

<img width="1792" alt="Screen Shot 2021-02-22 at 3 24 44 PM" src="https://user-images.githubusercontent.com/5672295/108702072-2ed54a00-7522-11eb-8082-6051ba6efc35.png">

<img width="1237" alt="Screen Shot 2021-02-18 at 6 51 04 PM" src="https://user-images.githubusercontent.com/5672295/108704184-174b9080-7525-11eb-95e9-deb794f43003.png">


Mobile first screen looks readable - someone wouldn't really get value out of using it on their phone as discussed so I only styled this screen.
<img width="612" alt="Screen Shot 2021-02-22 at 3 22 44 PM" src="https://user-images.githubusercontent.com/5672295/108702067-2e3cb380-7522-11eb-83c8-3c42e01f185a.png">

# Notes
- For some reason the geotiff I'm working with isn't showing all the info it should, so I can't style the table. 

# Testing
- `git pull`
- Click the About button to see the modal
- Close the modal
- Drag a geotiff onto the earth to see it change
- Note new styles on the following page.
- Cross browser

Closes #10, closes #9, closes #8, closes #14, closes #7, closes #6, closes #5
